tests/debugger/oegdb/oe_gdb_test.edl

```diff
@@ -2,11 +2,17 @@
 // Licensed under the MIT License.
 
 enclave {
+    enum magic_t {
+        MAGIC_VALUE=112233445566
+    };
+
     trusted {
         public int enc_add(int a, int b);
         public void enc_assert_debugger_binary_contract();
+	public void enc_test_stack_stitching();
```

Comment:
spacing looks off between lines 11 vs 12

Reply:
Good catch. Untabified the file.